src/main/java/org/cactoos/text/Strict.java

```diff
@@ -50,7 +50,7 @@ public Strict(final Pattern pattern, final Text origin) {
      * @param predicate The Func as a predicate
      * @param origin The Text
      */
-    public Strict(final Func<String, Boolean> predicate, final Text origin) {
+    public Strict(final Func<? super String, Boolean> predicate, final Text origin) {
```

Comment:
@DmitryBarskov It should be `CharSequence`

Reply:
@andreoss we still can pass params of type `Func<CharSequence, Boolean>` to this ctor.

Reply:
@DmitryBarskov please write a test to back this assertion, I'm not sure that's how it works, but I may be wrong :)